src/api/files/service.js

```diff
@@ -167,6 +291,26 @@ export async function persistFiles(files, persistedRetrievalKey) {
   }
 }
 
+/**
+ * Create a file in S3.
+ * @param {string} key - the key of the file
+ * @param {string} body - file body
+ * @param {string} contentType - content type
+ * @param {S3Client} client - S3 client
+ */
+async function createS3File(key, body, contentType, client) {
+  const result = await client.send(
```

Comment:
`await` can be dropped from here in favour of awaiting further up up the chain (as you are already doing)

Reply:
Resolved with 4e51c1f2a549f9688189c845bff781112e70c07b

Reply:
Ah I see. I do the same in a lot of places. Where it's a promise wrapping a promise I usually just inspect at the level above inside `createMain`. Good to know anyway :)

Reply:
Sound like we do the same thing. Here, because I'm not interested in the return value from `createS3File`, that [doesn't get assigned a variable](https://github.com/DEFRA/forms-submission-api/pull/120/files#diff-f879fff9dce2317507de8629238e236fafe196d025b54fadfa9eae79852a3805R80) either :upside_down_face: .